### PR TITLE
Docker ipfs config

### PR DESCRIPTION
**Pull request by bthaile**
changed webpack to create references to relative path instead of absolute path.
added run local environment variables for users to run augur ui without ipfs daemon running.
fixed issue with publishing, needed to $(...) the command
added clean up in docker container to remove ipfs install files

### CODE REVIEW

File: support/ipfs-crontab [hunk 1]
```diff
@@ -1 +1 @@
-* */12 * * * /usr/local/bin/ipfs name publish $(cat /augur/ipfs-deploy/NEW_BUILD_HASH) > /var/log/cron.log
+* */24 * * * /usr/local/bin/ipfs name publish $(cat /augur/ipfs-deploy/NEW_BUILD_HASH) > /var/log/cron.log
```
Comment (stephensprinkle) [inline support/ipfs-crontab]: Curious why the move to 24 over 12?

File: support/ipfs-run.sh [hunk 2]
```diff
@@ -1,13 +1,17 @@
 #!/bin/bash
 
-ipfs daemon &
-sleep 5s
+if [ "$RUN_LOCAL_ONLY" != "true" ]; then
+  ipfs daemon &
+  sleep 10s
```
Comment (stephensprinkle) [inline support/ipfs-run.sh]: Minor improvement — you could listen for the status of the ipfs process + pause the script.

File: webpack.config.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ let config = {
     filename: '[name].[chunkhash].js',
     chunkFilename: '[name].[chunkhash].js',
     path: PATHS.BUILD,
-    publicPath: '/'
+    publicPath: ''
```
Comment (stephensprinkle) [inline webpack.config.js]: 👍

File: support/ipfs-crontab (hunk 1 above)
Comment (bthaile) [reply support/ipfs-crontab]: I read that using a non default key the default time-to-live is 12 hours, but saw on the API the default is 24 hours, so I changed. Also the publish can throw up a Warning that gets spit out as a ERROR and I wanted to minimize frequency of the message.

File: support/ipfs-run.sh (hunk 2 above)
Comment (bthaile) [reply support/ipfs-run.sh]: I went to easy route and put in a sleep. I agree could have put in a do loop to process based on standard out.

File: webpack.config.js (hunk 3 above)
Comment (bthaile) [reply webpack.config.js]: yea, that was fun to find. learned a bit about webpack, cool stuff.

Comment (stephensprinkle) [reply webpack.config.js]: Nice!

File: support/ipfs-crontab (hunk 1 above)
Comment (stephensprinkle) [reply support/ipfs-crontab]: Yeah, defaults to 24, but It’s actually configurable as well.  

I’d originally set it to 12 hours to just make sure there was never a gap.

**Pull request merged**